include/odp/api/spec/event.h

```diff
@@ -93,6 +93,28 @@ int odp_event_type_multi(const odp_event_t event[], int num,
  */
 void *odp_event_user_area(odp_event_t event);
 
+/**
+ * Event user area and flag
+ *
+ * Returns pointer to the user area and outputs value of user flag associated
+ * with the event. The user area maps to the user area of underlying event type
+ * (e.g. odp_packet_user_area() for packets). If the event does not have user
+ * area, NULL is returned.
+ *
+ * The user flag maps to the user flag value of underlying event type (e.g.
+ * odp_packet_user_flag() for packets). If the event does not have user flag, a
+ * negative value is outputted.
```

Comment:
I guess alternatively this text could be more terse and just say that the function combines the two existing functions. More detailed specification then can be checked from the respective individual functions.